crates/vertigo/src/driver.rs

```diff
@@ -207,4 +210,47 @@ impl DomDriver {
         show_log(format!("fetch {:?} {}", method, url));
         self.driver.fetch(method, url, headers, body)
     }
+
+    pub fn get_location(&self) -> String {
```

Comment:
Maybe will be better that this function name is "get_hash_location" for consistency ?